aiogram/filters/command.py

```diff
@@ -78,6 +78,17 @@ def __init__(
                 " or their Iterable"
             )
 
+        prefixes = []
+        for char in prefix:
+            if char not in prefixes:
+                prefixes.append(char)
+
+        operations = {
+            BotCommand: lambda cmd, prefix: cmd.command.lstrip(prefix),
```

Comment:
You don't need to do anything with BotComand object because this case is impossible after implemented checks.